components/toast/stories/template.js

```diff
@@ -63,3 +64,74 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const ToastWrapOptions = ({
+	...args
+}, context ) => Container({
```

Comment:
```suggestion
export const ToastWrapOptions = (args, context ) => Container({
```

No need to deconstruct the args object here